README.md

```diff
@@ -0,0 +1,32 @@
+# Universal-Sierra-Compiler
+
+Universal-Sierra-Compiler is the tool for Sierra compilation. It compiles any ever-existing Sierra version to the CASM.
+
+## Usage
+
+To use the tool, just pass your Sierra json file path:
+
+```shell
+$ universal-sierra-compiler \
+  --sierra-input-path ./path/to/sierra.json
+  
+{"bytecode": ...}
+```
```

Comment:
I don't see a use case for that, isn't it better to just enforce passing output path?

Reply:
Nope, it would be useful for us. We can read the compiled sierra from the stdout and bypass creating and removing the file.